content/kubermatic/master/tutorials/12-audit-logs/_index.en.md

```diff
@@ -0,0 +1,59 @@
++++
+title = "Audit Logging"
+date = 2020-04-02T14:50:15+02:00
+weight = 120
++++
+
+Audit Logging is one of key security features provided by Kubernetes. Once enabled in kube-api, it provides a chronological record of operations performed on the clusters by users, administrators and other cluster components.
+
+Audit logging is also a key requirement of [Kubernetes CIS benchmark](https://www.cisecurity.org/benchmark/kubernetes/).
+
+For more details, you can can refer to the [upstream documentation](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/).
+
+{{% notice note %}}
+Kubernetes Audit Logging is introduced as a beta feature since **v1.11**.
```

Comment:
**was** introduced as a beta feature **in Kubernetes** v1.11.

Also, do we actually need this? We don't support anything <1.14.

Reply:
I am not sure. People are usually reluctant to upgrade. Are we sure we have no one running older versions ?